components/brave_wallet/browser/keyring_service.cc

```diff
@@ -722,13 +722,14 @@ void KeyringService::CreateWallet(const std::string& password,
 
   auto* keyring = CreateKeyring(mojom::kDefaultKeyringId, password);
   if (keyring) {
+    const std::string account_name = GetAccountName(1);
     const auto address =
-        AddAccountForKeyring(mojom::kDefaultKeyringId, GetAccountName(1));
+        AddAccountForKeyring(mojom::kDefaultKeyringId, account_name);
```

Comment:
why this change?

Reply:
https://github.com/brave/brave-core/pull/15707#discussion_r1024602143